### PR TITLE
Wire panel runtime fix

### PR DESCRIPTION
**Pull request by Alliostra**
## About The Pull Request

Wire panels without descriptions for wires in them will no longer runtime when interacted with.

## Why It's Good For The Game

fixes good

## Changelog
:cl:
fix: Wire panels without descriptions for wires in them will no longer runtime when interacted with.
/:cl:

### CODE REVIEW

File: code/datums/wires/wires.dm [hunk 1]
```diff
@@ -123,8 +123,9 @@ var/list/wireColours = list("red", "blue", "green", "darkred", "orange", "brown"
 	for(var/colour in wires)
 		html += "<tr>"
 		var/datum/wire_description/wd = get_description(GetIndex(colour))
-		if(user.stats && user.stats.getPerk(PERK_TECHNOMANCER) || user_skill && (wd.skill_level <= user_skill))
-			html += "<td[row_options1]><font color='[colour]'>[wd.description]</font></td>"
+		if(wd)
+			if(user.stats && user.stats.getPerk(PERK_TECHNOMANCER) || user_skill && (wd.skill_level <= user_skill))
+				html += "<td[row_options1]><font color='[colour]'>[wd.description]</font></td>"
```
Comment (SirRichardFrancis) [inline code/datums/wires/wires.dm:128]:
```suggestion
		if(wd && user.stats && user.stats.getPerk(PERK_TECHNOMANCER) || user_skill && (wd.skill_level <= user_skill))
			html += "<td[row_options1]><font color='[colour]'>[wd.description]</font></td>"
```

Comment (Alliostra) [reply code/datums/wires/wires.dm:128]: this will still runtime as "cannot read null.skill_level"

Comment (SirRichardFrancis) [reply code/datums/wires/wires.dm:128]: Could try `wd?.skill_level`, but whatever, PR is good as is.

Comment (Alliostra) [reply code/datums/wires/wires.dm:128]: `wd?.skill_level` also runtimes, byond moment I guess

**Review by SirRichardFrancis — APPROVED** [review]

**Review by TheShown911 — APPROVED** [review]

**Pull request merged**